Formula/liblastfm.rb

```diff
@@ -13,14 +14,24 @@ class Liblastfm < Formula
 
   depends_on "pkg-config" => :build
   depends_on "cmake" => :build
-  depends_on "qt"
+  depends_on "qt5"
   depends_on "fftw"
   depends_on "libsamplerate"
 
   def install
     mkdir "build" do
       system "cmake", "..", *std_cmake_args
       system "make", "install"
+      cd "tests" do
+        system "make"
+      end
+      share.install "tests"
     end
   end
+
+  test do
+    cp_r "#{share}/tests/.", testpath
+    assert_match "Totals: 3 passed, 0 failed, 0 skipped, 0 blacklisted, 0ms",
```

Comment:
`0ms` needs removed here as it's failing on Sierra (and generally is likely to be flaky). Is it not enough to rely on the exit-code alone here?